site/pages/react/quickstart.mdx

```diff
@@ -395,3 +395,67 @@ export default function Home() {
   );
 }
 ```
+
+## Other examples
```

Comment:
We probably should remove this, I didn't know where to put, then I found that the components where already being rendered